features/partners.feature

```diff
@@ -97,6 +98,7 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
       Dan heeft de persoon met burgerservicenummer '555550006' een partner met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 555550007 |
+      En heeft de partner met burgerservicenummer '555550007' de volgende 'naam' gegevens
```

Comment:
deze zin kan er toch tussenuit? Wat je hieronder noemt zijn gegevens direct op de partner, zeker geen 'naam' gegevens

Reply:
Opmerking klopt natuurlijkl. 
Ik vrees dat er hier met het oplossen van merge-conflict of de volgorde van mergen iets is misgegaan. Dit deel van het bestand wijkt behoorlijk af van wat ik in mijn branch heb staan.

Reply:
Gechecked, klopt in de nieuwe PR wel